src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/RadioGroup.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from 'react';
+import Radio from './Radio';
+
+type Props = {
+    value: string,
+    onChange?: () => void,
+    className?: string,
+    children?: React.ChildrenArray<*>,
```

Comment:
I am not completely sure what we should be doing here... Keep the flexibility we have with your approach, or build the label as child in the `Radio` component and therefore guarantee a consistent styling. /cc @chirimoya